css-responsive-layout/styles.css

```
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.fighter {
+  width: 100%;
+}
+
+.container {
+  max-width: 992px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col,
+.col-sm-half,
+.col-lg-third,
+.col-lg-fourth {
+  padding: 0 0.5rem;
+  flex-basis: 100%;
+}
```

All of your code should be in here. Like this
```suggestion
.col,
.col-sm-half,
.col-lg-third,
.col-lg-fourth {
  padding: 0 0.5rem;
  flex-basis: 100%;

  /* your code 👇 here */
  @media (width >= 576px) {
    &.col-sm-half {
      flex-basis: 50%;
    }
  }

  @media (width >= 768px) {
    &.col-lg-third {
      flex-basis: calc(100% / 3);
    }

    &.col-lg-fourth {
      flex-basis: 25%;
    }
  }
}
```

---

Hey Shawn, I already updated it. thanks for clarifying. I thought, i still have to include the ".row" class that is why i added it. also, i have an extra "}" earlier that i didn't noticed and should be move at the bottom."